programs/server/Server.cpp

```diff
@@ -870,12 +912,55 @@ if (ThreadFuzzer::instance().isEffective())
                     global_context->reloadZooKeeperIfChanged(config);
 
                 global_context->reloadAuxiliaryZooKeepersConfigIfChanged(config);
+
```

Comment:
We can probably make the code clearer if we add a new function named like `updateServers()`, and move the following code in there.